core/lib/spree/testing_support/calculator_helpers.rb

```diff
@@ -0,0 +1,5 @@
+shared_examples_for 'having a description' do
+  it "has a description" do
+    expect(subject.size).to be > 0
```

Comment:
I think expecting a `String` would be better. Every enumerable has a `size` method.